### ISSUE TITLE
other pretrained models 

### ISSUE THREAD

**Issue by iron-man45** [GitGyun/visual_token_matching#3]
appreciate your work. can you provide pretrained models of the TRAIN and the FINETUNE process . The "beit_base_patch16_224_pt22k.pth" is not enough for test .

**Comment by menghuaa** [comment]
> appreciate your work. can you provide pretrained models of the TRAIN and the FINETUNE process . The "beit_base_patch16_224_pt22k.pth" is not enough for test .

Can I add your contact information so that we can discuss this model？My wechat is menghuaaa123, my qq
is 2049304151

**Comment by GitGyun** [comment]
We have uploaded meta-trained and fine-tuned checkpoints.

**Comment by YDB04234** [comment]
Excuse me. I'm a novice. I'd like to ask whether the training process is still necessary when there are pre-trained models.

**Comment by GitGyun** [comment]
If you have a "meta-trained" model, you may skip the meta-training phase and go directly to the fine-tuning phase. However, if you only have pre-trained backbones such as BEiT, you should meta-train it with the matching module and the label decoder before fine-tuning.
